test/integration/sanSocialDominance.js

```diff
@@ -0,0 +1,59 @@
+const { testFieldTypes } = require('../helper.js')
+
+const {
+  testHistoricDataIsForbidden,
+  testHandlesNullData,
+  assertNumberOfRecords
+} = require('../integration_helper.js')
+
+const {
+  ethereumSlug,
+  from,
+  to,
+  source,
+  historicDataFrom,
+  historicDataTo,
+  days,
+  numberOfDays,
+  formatDate
+} = require('../setup.js')
+
+describe('SAN_SOCIAL_DOMINANCE', () => {
+  const expected = {
+    date: 'string',
+    dominance: 'number'
+  }
+
+  const response = san.SAN_SOCIAL_DOMINANCE(ethereumSlug, from, to, source)
+  const headers = response[0]
+  const results = response[1]
+
+  testFieldTypes(results, expected)
+  testHistoricDataIsForbidden(
+    san.SAN_SOCIAL_DOMINANCE,
+    ethereumSlug,
+    historicDataFrom,
+    historicDataTo,
+    source)
+  testHandlesNullData(
+    'fetchSocialDominance',
+    san.SAN_SOCIAL_DOMINANCE,
+    ethereumSlug,
+    from,
+    to,
+    source)
+
+  it('has proper headers', () => {
+    const expectedHeaders = ['Date', 'Dominance']
+    expect(headers).to.deep.equal(expectedHeaders)
+  })
+
+  it('returns a record per every day', () => {
+    const results = san.SAN_SOCIAL_DOMINANCE(ethereumSlug, from, to, source)
```

Comment:
Let's use santiment here

Reply:
Did as you asked and changed those ethereumSlugs with santiment slug